broadcaster/sequencenumbercatchupbuffer.go

```diff
@@ -29,11 +29,13 @@ type SequenceNumberCatchupBuffer struct {
 	messages     []*BroadcastFeedMessage
 	messageCount int32
 	limitCatchup func() bool
+	maxCatchup   func() int
```

Comment:
QQ: why is `maxCatchup` a `func() int` rather than just an `int`? I'm just curious what the advantage is

Reply:
This makes it hot reloadable